### ISSUE TITLE
indent Rule does not apply to object/array Destructuring

### ISSUE THREAD

**Issue by dropfen** [eslint/eslint#7975]
**Tell us about your environment**

* **ESLint Version:**
3.13.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default 
**Please show your full configuration:**

```
{ 
  indent: "error"
}
```

**What did you do? Please include the actual source code causing the issue.**

```js

    const {
        a,
      b,
    } = {};

    const [
        first,
      second,
    ] = [];



```

**What did you expect to happen?**

indentation errors

**What actually happened? Please include the actual, raw output from ESLint.**

nothing

**Comment by not-an-aardvark** [comment]
Thanks for the report.

This is a duplicate of https://github.com/eslint/eslint/issues/6813, so I'm going to close this issue. Please follow https://github.com/eslint/eslint/issues/6813 for updates instead.
